tests/mlflow/scripts/run_mlflow.sh

```
@@ -14,6 +20,14 @@ if [ -d "${VENV_DIR}/bin" ]; then
     export PATH="${MLFLOW_DIR}/${VENV_DIR}/bin:${PATH}"
 fi
 
+# for now we wipe out the mlflow db and mlruns directory
+rm -rf bin/mlflow*.db bin/mlruns
+if [ $? -ne 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)